README.md

```diff
@@ -2,11 +2,11 @@ Magpie
 ------
 
 Magpie contains a number of scripts for running Big Data software in
-HPC environments.  Thus far, Hadoop, Spark, Hbase, Storm, Pig, Mahout,
-Phoenix, Kafka, Zeppelin, and Zookeeper are supported.  It currently
-supports running over the parallel file system Lustre and running over
-any generic network filesytem.  There is scheduler/resource manager
-support for Slurm, Moab, Torque, and LSF.
+HPC environments.  Thus far, Hadoop, Spark, TensorFlow w/ & w/o Horovod,
+Hbase, Storm, Pig, Mahout, Phoenix, Kafka, Zeppelin, and Zookeeper are
+supported.  It currently supports running over the parallel file
+system Lustre and running over any generic network filesytem.  There
+is scheduler/resource manager support for Slurm, Moab, Torque, and LSF.
 
```

Comment:
Since it's not a tier 1 supported thing yet (and not with all scheduler + resource manager combos), lets just add a sentence "Experimental support for Tensorflow and Tensorflow with Horovod is also supported in some cases" after this paragraph.